### ISSUE TITLE
New PowerShell Preview, broken again

### ISSUE THREAD

**Issue by PrzemyslawKlys** [SeeminglyScience/EditorServicesCommandSuite#57]
Hi,

Sorry to report

```
Add-Type : Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\EditorServicesCommandSuite\1.0.0\EditorServicesCommandSuite.psm1:15 char:9
+         Add-Type -Path "$PSScriptRoot/EditorServicesCommandSuite.Edit ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-Type], ReflectionTypeLoadException
    + FullyQualifiedErrorId : System.Reflection.ReflectionTypeLoadException,Microsoft.PowerShell.Commands.AddTypeCommand

Import-Module : The module to process 'EditorServicesCommandSuite.psm1', listed in field 'ModuleToProcess/RootModule' of module manifest 'C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\EditorServicesCommand
Suite\1.0.0\EditorServicesCommandSuite.psd1' was not processed because no valid module was found in any module directory.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Microsoft.VSCode_profile.ps1:3 char:5
+     Import-Module -Name EditorServicesCommandSuite -Force
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (EditorServicesCommandSuite:String) [Import-Module], PSInvalidOperationException
    + FullyQualifiedErrorId : Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

**Comment by SeeminglyScience** [comment]
Thanks @PrzemyslawKlys!

Yeah I was warned about this one at least.  This is another omnisharp related breaking change, the last one was accidental, this one was purposeful but necessary.  Both cases should be exceedingly rare, just really bad luck on the timing.

Aiming to get another preview out this weekend with the fix.

**Comment by PrzemyslawKlys** [comment]
That's great! I refuse to use splat without your function :-p

**Comment by SeeminglyScience** [comment]
[Here's a preview build](https://artprodcus3.artifacts.visualstudio.com/A92124312-5ffc-43dc-a70b-5ed3c24753d2/72e7a1a1-a36b-4e39-b04e-268222d6d649/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL1NlZW1pbmdseVNjaWVuY2UwMzY4L3Byb2plY3RJZC83MmU3YTFhMS1hMzZiLTRlMzktYjA0ZS0yNjgyMjJkNmQ2NDkvYnVpbGRJZC81MC9hcnRpZmFjdE5hbWUvRWRpdG9yU2VydmljZXNDb21tYW5kU3VpdGUtdnMyMDE3LXdpbjIwMTY1/content?format=zip).  I'm going to hold off publishing it until the changes hit stable vscode-powershell.

I haven't tested with the preview-preview build they put in that thread though, so let me know if it breaks with it.

**Comment by PrzemyslawKlys** [comment]
```
Add-Type : Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\EditorServicesCommandSuite\1.0.0\EditorServicesCommandSuite.psm1:15 char:9
+         Add-Type -Path "$PSScriptRoot/EditorServicesCommandSuite.Edit ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-Type], ReflectionTypeLoadException
    + FullyQualifiedErrorId : System.Reflection.ReflectionTypeLoadException,Microsoft.PowerShell.Commands.AddTypeCommand

Import-Module : The module to process 'EditorServicesCommandSuite.psm1', listed in field 'ModuleToProcess/RootModule' of module manifest 'C:\Users\przem
yslaw.klys\Documents\WindowsPowerShell\Modules\EditorServicesCommandSuite\1.0.0\EditorServicesCommandSuite.psd1' was not processed because no valid modu
le was found in any module directory.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Microsoft.VSCode_profile.ps1:3 char:5
+     Import-Module -Name EditorServicesCommandSuite -Force
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (EditorServicesCommandSuite:String) [Import-Module], PSInvalidOperationException
    + FullyQualifiedErrorId : Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

It doesn't work with Preview of Preview :-)

**Comment by PrzemyslawKlys** [comment]
But you may be right that it's too unstable to put effort into it.

**Comment by SeeminglyScience** [comment]
If that's Windows PowerShell could you do me a favor and check if it works in PS7?

**Comment by PrzemyslawKlys** [comment]
It works, as in it loads, then when you do menu it works. But when you try splat it errors out

```
PS C:\Support\GitHub\Testimo> ParentContainsErrorRecordException: 
Line |
 351 |        $__cmdletization_objectModelWrapper.ProcessRecord($__cmdletizat …
     |        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "ProcessRecord" with "1" argument(s): "Could not load file or assembly 'OmniSharp.Extensions.LanguageProtocol, Version=0.17.0.0, Culture=neutral, PublicKeyToken=6d868dff454e6022'. The system cannot find the file specified."
```

**Comment by SeeminglyScience** [comment]
Thanks for checking that.

Yeah, I'm going to need to wait until that hits an actual release (even as a preview) to fix it.  Hopefully if we wait out this wave of breaks it won't be a problem anymore.  If not, I'll evaluate removing the dependency and rewriting all the logic myself.  Thanks for your patience!

**Comment by cjpalmer** [comment]
Has there been any movement on this? This is a great feature and I am hoping the dependent stuff as moved forward. This is very frustrating to not have this working.

**Comment by SeeminglyScience** [comment]
@cjpalmer Been a bit under the weather lately, sorry for the delay.  Thanks for the kick! It's definitely my top priority atm.

**Comment by cjpalmer** [comment]
You take care of yourself first. I will survive without easy splatting. We don't want to lose your contributions because you didn't take care of yourself.

**Comment by SeeminglyScience** [comment]
I'm still working on removing the dependencies that keep having issues, but it's taking longer than I expected.

So in the mean time, I've released another band-aid release.  Tested v1.0.0-beta3 on 5.1, 7.0.2, and 7.1-preview 3.  Hopefully this one lasts longer, but I'll keep working on rewriting the dependencies on the off chance it doesn't.

Thanks again to everyone for your patience and help with testing! ❤️
